### PR TITLE
Support for LFS 0.7E

### PR DESCRIPTION
**Pull request by neonsp**
Pull request to implement full support for LFS version 0.7E:

- Implement support for new IS_PLH packet related to player handicaps
- Add support to send TINY_MAL
- Make possible to send IS_MAL packets and implement reading car skins which was missing
- Add license type attribute to packet IS_NCI
- Implement SMALL_LCL - full control of lights including fog and extra lights
- Bump library version to 2.4.2.0

### CODE REVIEW

File: InSimDotNet/Packets/IS_NCI.cs [hunk 1]
```diff
@@ -32,6 +32,11 @@ public class IS_NCI : IPacket {
         /// Gets the language.
         /// </summary>
         public LfsLanguage Language { get; private set; }
+        
+        /// <summary>
+        /// Gets the language.
```
Comment (xspeedasx) [inline InSimDotNet/Packets/IS_NCI.cs]: language -> license

File: InSimDotNet/Packets/IS_MAL.cs [hunk 2]
```diff
@@ -13,8 +12,9 @@ namespace InSimDotNet.Packets
     /// You can set a list of up to 120 mods that are allowed to be used on a host
     /// Send zero to clear the list and allow all mods to be used
     /// </remarks>
-    public class IS_MAL : IPacket
+    public class IS_MAL : IPacket, ISendable
     {
+        public static int MAX_MAL_MODS = 120;
```
Comment (xspeedasx) [inline InSimDotNet/Packets/IS_MAL.cs]: this should probably be const or at least readonly

File: InSimDotNet/Packets/IS_PLH.cs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    /// <remarks>
+    /// Player handicaps
+    ///
+    /// Set handicaps per player.  These handicaps will remain until the player spectates or rejoins 
+    //  after returning from pits or garage (an IS_NPL will be sent in that case).
+    // 
+    /// </remarks>
+    public class IS_PLH : IPacket, ISendable
+    {
+        public static int PLH_MAX_PLAYERS = 40;
```
Comment (xspeedasx) [inline InSimDotNet/Packets/IS_PLH.cs]: this should probably be const or at least readonly

File: InSimDotNet/Packets/LocalCarLights.cs [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+        /// <summary>
+        /// Turn signals off
+        /// </summary>
+        public const int LCL_SIGNALS_OFF = LCL_SET_SIGNALS;
+
+        /// <summary>
+        /// Turn signal left
+        /// </summary>
+        public const int LCL_SIGNALS_LEFT = (1 << 16) + LCL_SET_SIGNALS;
```
Comment (xspeedasx) [inline InSimDotNet/Packets/LocalCarLights.cs]: suggestion: use `|` for bitwise flag operations (for all applicable flags)

**Review by xspeedasx — COMMENTED** [review]
Good job. Its nice to see some changes implemented. Although I commented on some nit-picks I will not request changes nor approve this PR. Imo it would benefit overall codebase to address those comments, but it would definitely work without changing anything.

File: InSimDotNet/Packets/IS_NCI.cs (hunk 1 above)
Comment (neonsp) [reply InSimDotNet/Packets/IS_NCI.cs]: Oh, good catch. Solved

File: InSimDotNet/Packets/IS_MAL.cs (hunk 2 above)
Comment (neonsp) [reply InSimDotNet/Packets/IS_MAL.cs]: I agree. Changed to const

File: InSimDotNet/Packets/LocalCarLights.cs (hunk 4 above)
Comment (neonsp) [reply InSimDotNet/Packets/LocalCarLights.cs]: I just followed the existing style, but I agree is better. Changed.

**Review by xspeedasx — APPROVED** [review]

**Comment by neonsp** [comment]
Thanks for the review! And thank you also for creating and keeping the version NET 6 alive.
I have been using this library for really long time so it was time to give it some love.

I have marked the LCS_SIGNALS and LCS_HEADLIGHTS deprecated and bumped version, so I think now is all good.

**Pull request merged**